bitnami/moodle/README.md

```diff
@@ -1,6 +1,6 @@
-# Moodle
+# Moodle(TM) LMS
```

Comment:
maybe we can use the `<sup>` syntax, instead:

```
# Moodle<sup>TM</sup> LMS
```

# Moodle<sup>TM</sup> LMS

Reply:
I am not sure if `sup` is supported in all Markdown parsers, though (it works in GitHub and GitLab)

Reply:
Thanks! done